rotkehlchen/rotkehlchen.py

```diff
@@ -510,7 +511,10 @@ def unlock_user(
         )
 
         self.migration_manager.maybe_migrate_data()
-        self.assets_updater = AssetsUpdater(self.msg_aggregator)
+        self.assets_updater = AssetsUpdater(
+            msg_aggregator=self.msg_aggregator,
+            globaldb=GlobalDBHandler(),
+        )
```

Comment:
```suggestion
        self.assets_updater = AssetsUpdater(
            msg_aggregator=self.msg_aggregator,
            globaldb=globaldb,
        )
```

Reply:
`globaldb` as a variable is not available here since this is called at `unlock()` and not at `__init__()`